src/views/parents/l10n.json

```diff
@@ -0,0 +1,26 @@
+{
+    "parents.introOne": "Scratch is a programming language and an online community where children can program and share interactive media such as stories, games, and animation with people from all over the world. As children create with Scratch, they learn to think creatively, work collaboratively, and reason systematically.",
+    "parents.introTwo": "Scratch is designed and maintained by the Lifelong Kindergarten group at the MIT Media Lab.",
+    "parents.introAbout": "About Scratch",
+    "parents.introEducators": "Info for educators",
```

Comment:
Perhaps you carried it over from the about page by accident?
![screenshot 2016-10-09 at 2 23 47 pm](https://cloud.githubusercontent.com/assets/12952370/19222660/02e80b9e-8e2c-11e6-8dab-22301086c4d1.png)